app/src/main/res/drawable/brush.xml

```diff
@@ -0,0 +1,5 @@
+<vector android:height="24dp" android:tint="?attr/iconFillColor"
```

Comment:
Rename all these to start with `ic_` it's common to prefix icons with that

Reply:
renamed all icons to follow convention